ext/pcntl/pcntl.c

```diff
@@ -53,6 +53,7 @@ typedef cpuset_t cpu_set_t;
 
 #if defined(HAVE_PTHREAD_SET_QOS_CLASS_SELF_NP)
 #include <pthread/qos.h>
+PHPAPI zend_class_entry *QosClass_ce;
```

Comment:
Why is this exposed via PHPAPI? Do we expect other extensions to reuse this enum?

Reply:
I do not think no :)